### PR TITLE
Guideconnect upgrade

### PR DESCRIPTION
**Pull request by MichaelKuzyk**

### CODE REVIEW

File: docs/payment-providers/interkassa/connect-manual.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        [![Step 3](images/interkassa-step3_1.png)](images/interkassa-step3_1.png)
+
+- Go to **_Paysystems_** and activate **at least 1** paysystem
+
+    ??? info "Screenshot"
+        [![Step 3](images/interkassa-step3_2.png)](images/interkassa-step3_2.png)
+
+- Go to **_Security_** and set up parameters:
+    - [x] Sign algorithm only **```SHA256```**
+    - [x] Require sign in payment form  **```Disabled```**
```
Comment (dzubchik) [inline docs/payment-providers/interkassa/connect-manual.md]: Must be enabled

File: docs/payment-providers/interkassa/connect-manual.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        [![Step 2](images/interkassa-step2_2.png)](images/interkassa-step2_2.png)
+
+#### Step 3: Configure Checkout
+
+- Open **_Checkout Settings_**
+
+    ??? info "Screenshot"
+        [![Step 3](images/interkassa-step3_1.png)](images/interkassa-step3_1.png)
+
+- Go to **_Paysystems_** and activate **at least 1** paysystem
```
Comment (dzubchik) [inline docs/payment-providers/interkassa/connect-manual.md]: Also the option unique id control must be enabled

File: docs/payment-providers/interkassa/connect-manual.md (hunk 1 above)
Comment (MichaelKuzyk) [reply docs/payment-providers/interkassa/connect-manual.md]: Fixed

File: docs/payment-providers/interkassa/connect-manual.md (hunk 2 above)
Comment (MichaelKuzyk) [reply docs/payment-providers/interkassa/connect-manual.md]: Fixed

**Pull request merged**